### PR TITLE
Update graceful shutdown documentation

### PR DESCRIPTION
**Pull request by metanet**

### CODE REVIEW

File: src/FAQ.md [hunk 1]
```diff
@@ -273,24 +273,36 @@ Client-server topology fits better if there are multiple applications sharing th
 
 ## How do I know it is safe to kill the second member?
 
-Below code snippet shuts down the cluster members if the cluster is safe for a member shutdown.
+Starting with Hazelcast 3.7, graceful shutdown of a hazelcast member can be initiated any time as follows:  
+
```
Comment (Serdaro) [inline src/FAQ.md]: ....of a Hazelcast member can be......

File: src/FAQ.md [hunk 2]
```diff
@@ -273,24 +273,36 @@ Client-server topology fits better if there are multiple applications sharing th
...
+```java
+  hazelcastInstance.shutdown(); 
+```
+
+Once a hazelcast member initiates a graceful shutdown, data of the shutting down member is migrated to the other nodes automatically.
+
```
Comment (Serdaro) [inline src/FAQ.md]: Once a Hazelcast member initiates...........to the other members automatically.

File: src/FAQ.md [hunk 3]
```diff
@@ -273,24 +273,36 @@ Client-server topology fits better if there are multiple applications sharing th
...
+However, there is no such guarantee for termination.
+
+Below code snippet terminates a member if the cluster is safe. Safe state means; there are no partitions being migrated and all backups are in sync when this method is called.
```
Comment (Serdaro) [inline src/FAQ.md]: .....is safe. Safe state means that there are no partitions.....

File: src/FAQ.md [hunk 4]
```diff
@@ -273,24 +273,36 @@ Client-server topology fits better if there are multiple applications sharing th
...
 ```java
 PartitionService partitionService = hazelcastInstance.getPartitionService();
 if (partitionService.isClusterSafe()) {
-  hazelcastInstance.shutdown(); // or terminate
+  hazelcastInstance.getLifecycleService().terminate(); 
 }
 ```
 
-Below code snippet shuts down the local member if the member is safe to be shutdown.
+Below code snippet terminates the local member if the member is safe to terminate. Safe state means; all backups of partitions currently owned by local member are in sync when this method is called.
```
Comment (Serdaro) [inline src/FAQ.md]: ....terminate. Safe state means that all backups of the partitions currently.......

File: src/FAQ.md [hunk 5]
```diff
@@ -273,24 +273,36 @@ Client-server topology fits better if there are multiple applications sharing th
...
 
 ```java
 PartitionService partitionService = hazelcastInstance.getPartitionService();
 if (partitionService.isLocalMemberSafe()) {
-  hazelcastInstance.shutdown(); // or terminate
+  hazelcastInstance.getLifecycleService().terminate();
 }
 ```
 
+Please keep in mind that 2 code snippets shown above are inherently racy. If member failures occur in the cluster after the safety condition check passes, termination of the local member can lead to data loss. For safety of the data, graceful shutdown API is highly recommended.
```
Comment (Serdaro) [inline src/FAQ.md]: ......that two code snippets shown....

**Comment by metanet** [comment]
Updated the pr.

**Pull request merged**